test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -362,6 +372,7 @@ public void createParticipantEmailEnabledNoVerificationWanted() {
         
         verify(accountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
         verify(account).setStatus(AccountStatus.ENABLED);
+        verify(account).setEmailVerified(Boolean.TRUE);
```

Comment:
We test that emailVerified is set to true here, but we don't test that emailVerified is not set to true in the createParticipantEmailEnabledVerificationWanted(). We probably should, to make sure that if branch is properly tested.